action/remove.go

```diff
@@ -2,24 +2,70 @@ package action
 
 import (
 	"os"
+	"os/exec"
 	"path/filepath"
+	"strings"
 
 	"github.com/helm/helm/log"
+	"github.com/helm/helm/manifest"
 )
 
+// kubeGet executes the kubectl command, override in tests
+var kubeGet func(string) string
```

Comment:
@adamreese I was told by @technosophos that you were working on pulling out kubectl functionality into an interface, which would provide a good way to mock as you said. do you have that work somewhere in a PR?